osf/management/commands/spam_metrics.py

```diff
@@ -0,0 +1,113 @@
+import csv
+import pytz
+import tempfile
+import datetime
+import logging
+import requests
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from framework.celery_tasks import app as celery_app
+from osf.models import NodeLog, OSFUser, SpamStatus, PreprintLog
+
+from website import settings
+
+logger = logging.getLogger(__name__)
+
+
+@celery_app.task(name='osf.management.commands.spam_metrics')
+def gather_metrics(dry_run=False):
+    today = timezone.now().today()
+    first = today.replace(day=1)
+    last_month = first - datetime.timedelta(days=1)
+    last_month.replace(tzinfo=pytz.utc)
```

Comment:
If this runs on the first of each month, e.g. `2-1-2023`, `last_month` would be `1-31-2023` at runtime, which I don't imagine would give a very accurate sample for January